CHANGELOG.rst

```diff
@@ -17,20 +17,154 @@ Changes to "0.14.0"
+- Redesign of public class structure
+
+  - Apply frontend redesign to Container and deriving classes #1115 #1159
+- Removal of APIs
+  - Dataset::transform, Dataset::compression and Dataset::chunksize #1043
 
 Other
 """""
-- Catch2: updated to 2.13.10 #1299 #...
+- Catch2: updated to 2.13.10 #1299 #1344
+- Tests & Examples:
+
+  - Test: Interleaved Write and Close #1073 #1078
+  - Extend and fix examples 8a and 8b (bench write/read parallel) #1131 #1144 #1231 #1359
+  - GPU support in example 8a #1240
+  - Extensive Python example for Streaming API #1141
+  - General overhaul of examples to newest API standards #1371
+- CI
+
+  - URL Check for broken links #1086
+  - CI savings (abort prior push, draft skips most) #1116
+  - Appveyor fixes for Python Executable #1127
+  - Pre-commit and clang-format #1142 #1175 #1178 #1032 #1222 #1370
+  - ADIOS1: Fix Serial Builds, CI: Clang 10->12 #1167
+  - Upgrade NVHPC Apt repository #1241
+  - Spack upgrade to v0.17.1 and further fixes #1244
+  - Update CUDA repository key #1256
+  - Switch from Conda to Mamba #1261
+  - Remove ``-Wno-deprecated-declarations`` where possible #1246
+  - Expand read-only permission tests #1272
+  - Ensure that the CI also build against ADIOS2 v2.7.1 #1271
+  - Build(deps): Bump s-weigand/setup-conda from 1.1.0 to 1.1.1 #1284
+  - Style w/ Ubuntu 22.04 #1346
+  - Add CodeQL workflow for GitHub code scanning #1345
+  - Cache Action v3 #1358 #1362
+- CMake
+
+  - Extra CMake Arg Control in setup.py #1199
+  - Do not strip Python symbols in Debug #1219
+  - Disable in-source builds #1079
+  - Fixes for NVCC #1102 ##1184
+  - Set RPATHs on installed targets #1105 #1103
+  - CMake 3.22+: Policy CMP0127 #1165
+  - Warning Flags First in CXXFLAGS #1172
+- Docs
+
+  - More easily findable documentation for ``-DPython_EXECUTABLE`` #1104 and lazy parsing #1111
+  - HDF5 performance tuning and known issues #1129 #1132
+  - HDF5: Document HDF5_USE_FILE_LOCKING #1106
+  - SST/libfabric installation notes for Cray systems #1134
+  - OMPI_MCA_io Control #1114
+  - Update Citation & Add BibTeX (#1168)
+  - Fix CLI Highlighting #1171
+  - HDF5 versions that support collective metadata #1250
+  - Recommend Static Build for Superbuilds #1325
+  - Latest Sphinx, Docutils, RTD #1341
+- Tooling
+  - openpmd-pipe: better optional support for MPI #1186 #1336
+- Enable use of Series::setName() and Series::setIterationEncoding() in combination with file-based encoding 1081
+- Remove DATATYPE, HIGHEST_DATATYPE AND LOWEST_DATATYPE from Datatype enumeration #1100
+- Check for undefined datatypes in dataset definitions #1099
+- Include StringManip header into public headers #1124
+- Add default constructor for DynamicMemoryView class #1156
+- Helpful error message upon wrong backend specification #1214
+- Helpful error message for errors in ``loadChunk`` API #1373
+- No warning when opening a single file of a file-based Series #1368
+- Add ``IterationIndex_t`` type alias #1285
 
 
 0.14.3
```

Comment:
I used the git log between `0.14.0` and `dev` as a basis for this, so this will rather be a merge.
Most important is probably a more detailed listing of the HDF5 performance optimizations.

Reply:
done